src/import/generic/memory/lib/utils/shared/mss_generic_consts.H

```diff
@@ -230,6 +230,8 @@ enum dimm_type
 {
     RDIMM = 0b0001,
     LRDIMM = 0b0100,
+    SORDIMM = 0b1000,
+    MINIRDIMM = 0b1001,
```

Comment:
MINIRDIMM is 0b0101 according to the JEDEC SPD spec, not 0b1001 - which is a SO-UDIMM.

Reply:
Thanks for catching this. I've updated the code to be correct.